### DISCUSSION TITLE
Delete Patches - make check dependencies optional

### DISCUSSION THREAD

**Discussion by wirmachenbunt** [cables-gl/cables_docs#488] [Feature Requests] [answered]
since it takes ages for the check, it would be nice if this well intented check, could be made optional.

and it would be nice to have a direct link / icon to delete patches
![image](https://user-images.githubusercontent.com/1138673/219962826-844c4c02-6094-412b-8cd5-8e5d27001a85.png)

this would render cleaning up the patch library a little faster

**Comment by pandrr** [comment, marked as answer]
currently optimizing this, on dev its already faster

no trash icon there, no space... thinking of an three-dots menu wich multiple options

**Reply by wirmachenbunt** [reply]
ok, cool. any way to speed up deleting is appreciated. i like my lib tidy ;)
